data/builtInFeatures.js

```diff
@@ -0,0 +1,120 @@
+// https://github.com/zloirock/core-js
+
+/* eslint-disable quotes */
```

Comment:
You can just set it with `avoidEscape`

http://eslint.org/docs/rules/quotes#options